generate.2.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-BASE=$(pwd)
-
-while getopts g:a:t:p:o:v:m:u: option
+while getopts g:a:t:p:o:v:m:u:w: option
```

Comment:
`pip install` and virtualenv are not required in Heroku. I've removed them in the PR